md/applicationCreation.md

```diff
@@ -0,0 +1,71 @@
+# Application Descriptor
+
+
+
+## Overview
+
+In Bonita BPM 7.5, we introduced the possibility to manage application descriptors from the Studio.  
+It accounts for an additional step to ease the management of [Living Applications](applications.md) in Bonita BPM, since our objective is to offer a unique application deployment artifact, managed from the Studio.  
+Therefore, all resources used by applications will be centralized and shared by the Bonita BPM development suite: Bonita BPM Studio and the UI Designe.  
+Those resources are used to create Application Descriptors from the Studio. Application Descriptors can be deployed in the Portal to create Applications.  
+
+## Definition 
+
+Basically, an **application descriptor** is an XML file. It represents the skeleton of an Application, as it contains the references to all resources installed in Bonita BPM Portal and used by a given application.  
```

Comment:
'as it contains the references to all resources installed in Bonita BPM Portal ' 

the *all* resources part is wrong. (no rest api extension by example)